tests/foreman/cli/test_remoteexecution.py

```diff
@@ -531,18 +532,22 @@ def test_positive_run_receptor_installer(self, default_sat):
             in ["succeeded", "failed"],
             timeout="1500s",
         )
+
+        result = JobInvocation.get_output({'id': invocation_id, 'host': default_sat.hostname})
+        logger.debug(f"Invocation output>>\n{result}\n<<End of invocation output")
+        # if installation fails, it's often due to missing rhscl repo -> print enabled repos
+        repolist = default_sat.execute('yum repolist')
+        logger.debug(f"Repolist>>\n{repolist}\n<<End of repolist")
```

Comment:
Is this kept for debugging purposes or intentional?

If it's intentional, then I think this is not needed and its also adds to test execution time!

Reply:
From the OP:
```
Also, it's necessary to output debugging information such as repolist
and invocation output to determine why this fails in the automation.
```

Reply:
@lhellebr If you are willing to know that for CI then that's fine but you can always get that output if u run it locally.

Reply:
That way, I can get a different output since the automation environment can be different. My use case: I need to verify that in automation, there is a certain repository missing.

Anyway, it's good to see the result of a failure in the log when investigating automation failures - that's what the logs are for. The job log isn't available currently.

Reply:
Sure! I understand the concern!